public/static/js/build.js

```diff
@@ -77,22 +77,26 @@ mciModule.controller('BuildViewController', function($scope, $http, $timeout, $r
   $scope.subscriptions = [];
   $scope.hide_add_subscription = true;
   $scope.triggers = [
-      // TODO uncomment as we implement these
-      // {
-      //    trigger: "failure",
-      //    resource_type: "BUILD",
-      //    label: "any task in this build fails",
-      // },
-      // {
-      //    trigger: "first-failure-by-variant",
-      //    resource_type: "BUILD",
-      //    label: "the first task in this build fails",
-      // },
-      // {
-      //    trigger: "regression",
-      //    resource_type: "BUILD",
-      //    label: "a previously passing task fails",
-      // },
+    {
```

Comment:
also include success, given today's conversation? same question for other JS files as well

Reply:
Yep, will update this. Anything other than this?

Reply:
just that, otherwise lgtm